ner_v2/tests/pattern/phone_number/en/test_phone_number_detection.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+
+from django.test import TestCase
+
+from ner_v2.detectors.pattern.phone_number.phone_number_detection import PhoneDetector
+
+
+class PhoneDetectionTest(TestCase):
```

Comment:
This class can be written agnostic of language. There should be a csv file with columns, 'language', 'message', 'phone_numbers (pipe separated)', 'Original text (pipe separated)', 'Description'. If I want to add more test cases, I should not be writing a function every time. I should just be adding a row. Language is not needed but just for accountability
For example - 
'en', 'call me at 07926645075 from +91 9833430535', '07926645074 | 919833430535', '07926645075 | +91 9833430535'

Reply:
Done